WooCommerce/Classes/ViewRelated/Products/UpdateInventory/UpdateProductInventoryViewModel.swift

```diff
@@ -180,9 +217,17 @@ private extension UpdateProductInventoryViewModel {
         isPrimaryButtonLoading = true
 
         // TODO: Handle error
-        try? await inventoryItem.updateStockQuantity(with: newQuantity, stores: stores)
+        do {
+            inventoryItem = try await inventoryItem.updateStockQuantity(with: newQuantity, stores: stores)
 
-        isPrimaryButtonLoading = false
-        updateQuantityButtonMode = .increaseOnce
+            isPrimaryButtonLoading = false
+            updateQuantityButtonMode = .increaseOnce
+        }
+        catch {}
```

Comment:
Do we log the error here for the moment? Or we'll handle this differently?

Reply:
The PR handling the error is coming, so it will be handled there.